CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.27
+
+<!-- generated comparing v5.0.0-alpha.26..next -->
+
+_Mar 5, 2021_
+
+Big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 👩‍🎤 Convert 8 components to emotion (#25091, #25158, #25146, #25142, #25166) @natac13, @mngu, @m4theushw, @praveenkumar-kalidass.
+- 📚 Convert 5 components demos to emotion (#25164, #25183, #25180, #25145, #25138) @vicasas
+- And many more 🐛 bug fixes and 📚 improvements.
+
+### `@material-ui/core@5.0.0-alpha.27`
+
+- <!-- 13 --> [Drawer] Migrate to emotion (#25091) @natac13
+- <!-- 20 --> [LinearProgress] Migrate to emotion (#25158) @mngu
+- <!-- 06 --> [Pagination] Migrate Pagination and PaginationItem to emotion (#25146) @mngu
+- <!-- 21 --> [Radio] Migrate to emotion (#25152) @mngu
+- <!-- 10 --> [Snackbar] Migrate to emotion (#25142) @m4theushw
+- <!-- 25 --> [SpeedDial] Migrate to emotion (#25166) @m4theushw
+- <!-- 12 --> [Stepper] Migrate StepConnector to emotion (#25092) @praveenkumar-kalidass
+- <!-- 16 --> [Autocomplete] Support readonly type for the options (#25155) @silver-snoopy
+- <!-- 27 --> [Table] Use primary cover over secondary for selected state (#25182) @beaudry
+- <!-- 08 --> [Table] Remove default role logic in TableCell (#25105) @silver-snoopy
+- <!-- 23 --> [theme] Fix styleOverrides with nested selectors (#25156) @ruppysuppy
+- <!-- 07 --> [styled] Fix override logic to support component without root (#25143) @niting143
```

Comment:
Did we decide how we want to order them (date vs alphanumeric)? Right now this looks like it's neither one (check Stepper, Autocomplete, Table, Table where the last Table change happened before the first table change)

Reply:
I've ordered them alphabetically, I can change the order if necessary. Does the comment in the front reflect the order of which the PR was merged?

Reply:
> Does the comment in the front reflect the order of which the PR was merged?

At a high level it represents how items should be ordered in their sections. Right now this is by date but if we think it should be alphabetically then we can change the script that generates the changelog.

The goal is to have as few manually editing as possible to avoid mistakes (such as the one before the change) and have the changelog consistent which is important to easily scan the  changelog. If developers can't easily scan the changelog for changes that might be affecting them, then they're less likely to upgrade. Changing the formatting generally or ordering specifically makes it harder to scan.

Reply:
That makes sense. I'd say alphabetical makes more sense, as it is naturally easier to scan. Developers don't know the order of when something was merged, so I think that's why the alphabetical is better. Let's leave this open to get opinions from more people.

Reply:
+1 for alphabetical, as changes are all released at once